src/App.jsx

```diff
@@ -1,44 +1,38 @@
 import React from 'react';
-import 'bulma/css/bulma.css';
-import '@fortawesome/fontawesome-free/css/all.css';
-import './App.scss';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Comment:
The current logic assumes that the `tabs` array will always contain at least one element. If the array is empty, `tabs[0]` will be `undefined`, leading to potential errors when accessing properties of `activeTab`. Consider adding a check to handle the scenario where `tabs` is empty, such as returning `null` or a default message when there are no tabs.